### PR TITLE
Update calll to deprecated method

### PR DESCRIPTION
**Pull request by WhyNotHugo**

### CODE REVIEW

File: src/frame-writer.cpp [hunk 1]
```diff
@@ -654,8 +654,7 @@ bool FrameWriter::add_frame(const uint8_t* pixels, int64_t usec, bool y_invert)
         filtered_frame->pict_type = AV_PICTURE_TYPE_NONE;
 
         // So we have a frame. Encode it!
-        AVPacket pkt;
-        av_init_packet(&pkt);
+        AVPacket pkt = *av_packet_alloc();
```
Comment (ammen99) [inline src/frame-writer.cpp:657]: This seems as if we are leaking the packet. Isn't that true?

**Comment by WhyNotHugo** [comment]
I believe you are correct, I will review this.

**Comment by ammen99** [comment]
The same changes have been proposed and also merged in #193, but thank you for the PR nonetheless :)

**Comment by WhyNotHugo** [comment]
Apologies for never getting back.

Glad that this has made it in anyway, cheers!

**Pull request closed without merging**